modules/persona/src/test/ts/atomic/DefaultAvatarTest.ts

```diff
@@ -1,7 +1,7 @@
+import { context, describe, it } from '@ephox/bedrock-client';
 import { Optional, Fun } from '@ephox/katamari';
-import { describe, expect, it } from 'vitest';
-
-import * as DefaultAvatar from '../../../../main/ts/defaultavatar/DefaultAvatar.ts';
+import { assert } from 'chai';
```

Comment:
chai does have `expect`, if that's the style of test you'd rather write 🤷‍♂️ 

personally I stick to bedrock assert but we don't really have a fixed standard for this anymore.